### PR TITLE
SDN-3635: OSDN kube 1.26.0 rebase 2nd attempt

### PR DESCRIPTION
**Pull request by msherif1234**

### CODE REVIEW

File: pkg/cmd/openshift-sdn-node/kube_proxy.go [hunk 1]
```diff
@@ -121,6 +118,8 @@ func newProxyServer(config *kubeproxyconfig.KubeProxyConfiguration, client clien
 	klog.V(0).Infof("kube-proxy running in single-stack %s mode", iptInterface.Protocol())
 
 	if proxyMode == proxyModeIPTables {
+		// userspace proxy mode was removed upstream in 1.26,
+		// let's fallback to iptables
```
Comment (danwinship) [inline pkg/cmd/openshift-sdn-node/kube_proxy.go:122]: stray comment

**Comment by danwinship** [comment]
commit message is still wrong too:

> Also, k8s.io/kubernetes/pkg/proxy/userspace was removed from 1.26 code base, so let's keep  "--proxy-mode userspace" for backwards compatibility, but let's make it use iptables instead.

File: pkg/cmd/openshift-sdn-node/kube_proxy.go (hunk 1 above)
Comment (msherif1234) [reply pkg/cmd/openshift-sdn-node/kube_proxy.go]: removed

**Comment by msherif1234** [comment]
azure job is passing and no sign of endpoint issue for openshit-console

**Pull request merged**